stl/CMakeLists.txt

```diff
@@ -440,13 +445,16 @@ function(add_stl_dlls D_SUFFIX THIS_CONFIG_DEFINITIONS THIS_CONFIG_COMPILE_OPTIO
     target_compile_definitions(msvcp${D_SUFFIX}_implib_objects PRIVATE "_DLL;${THIS_CONFIG_DEFINITIONS}")
     target_compile_options(msvcp${D_SUFFIX}_implib_objects PRIVATE "${THIS_CONFIG_COMPILE_OPTIONS};/EHsc") # No /GL!
 
+    add_library(msvcp${D_SUFFIX}_satellite_dll_objects OBJECT ${SATELLITE_DLL_SOURCES})

```

Comment:
Should this have a `target_compile_definitions` saying `"_DLL;${THIS_CONFIG_DEFINITIONS}"`? If not, why not?

Reply:
It really doesn't matter; there is nothing in that file which inspects `_DLL` (note that the .obj doesn't care about the IDL setting here). Do you want it added for consistency?

Reply:
I think it would be nice to add for consistency. The concern is that during maintenance/extension, if everything is consistent, then copy-pasting will be more likely to work. If it were important to omit something, then it should be commented as omitted (e.g. the "no /GL" comment elsewhere).

Unlike product code, a lot of quasi-informed copy-pasting happens in build scripts.